### PR TITLE
feat: updated remote UI with transport and digit buttons

### PR DESCRIPTION
**Pull request by martonborzak**

### CODE REVIEW

File: components/remote/ui/ir/CardButtons.qml [hunk 1]
```diff
@@ -69,44 +65,121 @@ Item {
...
+    //    Button {
+    //        visible: obj.isSupported(Remote.F_GUIDE)
+    //        anchors.bottom: parent.bottom
+    //        anchors.right: parent.right
+    //        title: qsTr("Guide") + translateHandler.emptyString
+    //        mouseArea.onClicked: {
+    //            obj.guide();
+    //        }
+    //    }
+
```
Comment (zehnm) [inline components/remote/ui/ir/CardButtons.qml]: What's with the commented out buttons?

**Review by zehnm — APPROVED** [review]
Have you thought about custom layouts and dynamic button placements?
It looks like a hardcoded layout with all supported buttons shown.
I would really like to customize it and hide certain buttons I'll never use.

File: components/remote/ui/ir/CardButtons.qml (hunk 1 above)
Comment (martonborzak) [reply components/remote/ui/ir/CardButtons.qml]: opps, my bad :)

**Comment by martonborzak** [comment]
> Have you thought about custom layouts and dynamic button placements?
> It looks like a hardcoded layout with all supported buttons shown.
> I would really like to customize it and hide certain buttons I'll never use.

I thought about this as a first step. The buttons won't show up if there's no supported feature defined. It's kind of customizable as well at this stage. But a fully dynamic custom layout would be next step I think.

**Comment by zehnm** [comment]
> I thought about this as a first step. The buttons won't show up if there's no supported feature defined. It's kind of customizable as well at this stage. But a fully dynamic custom layout would be next step I think.

Good point, first it needs to work, then we make it awesome 🥇
This will make many people happy, especially the ex-neeo crowd :-)

**Comment by martonborzak** [comment]
Exactly, one step at a time :)

**Pull request merged**